src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizardController.js

```diff
@@ -97,7 +98,22 @@
             } else {
                 model.getBatchMetadata().showError(possibleError);
             }
+        } else if (task === 'showAdvanced') {
```

Comment:
Is it possible to break this out into using another function? This one gets a lot of noise already because it's also listening for events from the footer (definitely something we'll change with the rewrite!). Maybe change these 3 functions to one that's handleProcessingSettingsChange?